packages/cli/package.json

```diff
@@ -49,7 +49,7 @@
     "access": "public"
   },
   "dependencies": {
-    "@graphql-mesh/cli": "0.66.2",
+    "@graphql-mesh/cli": "0.67.0-alpha-e0a33e998.0",
```

Comment:
Let's try to get this to stable version before merging here

Reply:
Done!